nessie/src/main/java/org/apache/iceberg/nessie/NessieTableOperations.java

```diff
@@ -146,6 +147,28 @@ protected void doRefresh() {
     refreshFromMetadataLocation(metadataLocation, null, 2, l -> loadTableMetadata(l, reference));
   }
 
+  private CommitMultipleOperationsBuilder makeNessieCommit(
+      IcebergTable newTable, boolean addNamespaces) {
+    CommitMultipleOperationsBuilder commit =
+        client
+            .getApi()
+            .commitMultipleOperations()
+            .operation(Operation.Put.of(key, newTable, table));
+
+    if (addNamespaces) {
```

Comment:
why do we want to make this new behavior the default? I think there are a few systems (trino/presto) that actually rely on the behavior where a namespaces must exist before creating tables in it

Reply:
Interesting. So, even though Nessie supports implicit namespaces, the engine (Presto, Trino) has a restriction that it should exist?
With Spark, hive/glue catalog also expects a namespace to exist before creating a table. But optional for Nessie. 

Nessie's behaviour with spark and presto/trino is different now!